NPE when Croissant isn't enabled/configured

We upgraded to v6.5, and server.log has a ton of

```
[2025-04-17T18:13:31.298+0000] [Payara 6.2024.6] [SEVERE] [] [jakarta.enterprise.resource.webcontainer.faces.context] [tid: _ThreadID=105 _ThreadName=http-thread-pool::jk-connector(5)] [timeMillis: 1744913611298] [levelValue: 1000] [[
  jakarta.faces.view.facelets.TagAttributeException: /dataset.xhtml @75,75 test="#{empty DatasetPage.croissant}" /dataset.xhtml @75,75 test="#{empty DatasetPage.croissant}": java.lang.NullPointerException
jakarta.faces.view.facelets.TagAttributeException: /dataset.xhtml @75,75 test="#{empty DatasetPage.croissant}" /dataset.xhtml @75,75 test="#{empty DatasetPage.croissant}": java.lang.NullPointerException
<snip>
Caused by: jakarta.el.ELException: /dataset.xhtml @75,75 test="#{empty DatasetPage.croissant}": java.lang.NullPointerException
        at com.sun.faces.facelets.el.TagValueExpression.getValue(TagValueExpression.java:77)
        at com.sun.faces.facelets.tag.TagAttributeImpl.getObject(TagAttributeImpl.java:289)
        ... 79 more
Caused by: jakarta.el.ELException: java.lang.NullPointerException
```

It is my understanding that Croissant must be specifically enabled, so this may simply be log noise, but it's generating quite a bit of noise.

--- a/src/main/webapp/dataset.xhtml
+++ b/src/main/webapp/dataset.xhtml
@@ -71,14 +71,12 @@
             <ui:define name="jsonld_header">
                 <ui:fragment rendered="#{!DatasetPage.anonymizedAccess}">
                     <script type="application/ld+json">
-                        <c:choose>
-                            <c:when test="#{empty DatasetPage.croissant}">
-                                <h:outputText value="#{DatasetPage.jsonLd}"/>
-                            </c:when>
-                            <c:otherwise>
-                                <h:outputText value="#{DatasetPage.croissant}"/>
-                            </c:otherwise>
-                        </c:choose>
+                        <ui:fragment rendered="#{DatasetPage.croissant == null}">
+                            <h:outputText value="#{DatasetPage.jsonLd}"/>
+                        </ui:fragment>
+                        <ui:fragment rendered="#{DatasetPage.croissant != null}">
+                            <h:outputText value="#{DatasetPage.croissant}"/>
+                        </ui:fragment>
                     </script>
                 </ui:fragment>
             </ui:define>

